tools/README.md

```diff
@@ -5,34 +5,64 @@ This tool allows ballerina message broker developers to get an idea about the pe
 - Connection Load  - The number of message producers , or the number of concurrent connections a system can support.
 - Message throughput - The number of messages or message bytes that can be pumped through a messaging system per second.
 
-At the moment the tool allows developers to publish messages to queues and test the performance and get a report of the result.
+At the moment the tool allows developers to publish/consume messages to/from queues and topics and test the performance and get a report of the result.
 
-## How to use 
+# How to use
 
-1. Copy following jars to JMETER_HOME/lib.
+## Prerequisite
+
+1. Create two queues ```micro_benchmark_queue1``` and ```micro_benchmark_queue2```.The script uses these queues separately to study the performance of queue and topic operations.
```

Comment:
Is there a way to skip this step? May be by creating a queue and a binding using the rest api?